pkg/scheduler/api/node_info.go

```diff
@@ -176,12 +188,16 @@ func (ni *NodeInfo) AddTask(task *TaskInfo) error {
 	if ni.Node != nil {
 		switch ti.Status {
 		case Releasing:
+			if err := ni.allocateIdleResource(ti); err != nil {
+				return err
+			}
 			ni.Releasing.Add(ti.Resreq)
-			ni.Idle.Sub(ti.Resreq)
 		case Pipelined:
 			ni.Releasing.Sub(ti.Resreq)
```

Comment:
Does this face the same problem?

Reply:
Yes, So I deleted this line and replace it to line 191-193.

Reply:
> @william-wang Could you add more comments why this can cause a panic?
In customer's environment, when the GPU was lost. the ni.Idle decreased and  the value was less than ti.Resreq. Then the panic happened in Sub func. So we need to add a judgement before the execution of ni.Idle.Sub(ti.Resreq).